### PR TITLE
Fixed cookie notice text and privacy policy link.

### PR DESCRIPTION
**Pull request by juanlopez4691**
### Ticket
- [core-standards: Update cookie notice text](https://app.asana.com/0/30975952354191/1124673186025759)

### CODE REVIEW

File: languages/core-standards-de_DE.po [hunk 1]
```diff
@@ -116,3 +115,13 @@ msgstr ""
...
+
+#~ msgid ""
+#~ "This website uses cookies to give you the best possible service. By "
+#~ "continuing to use the website, you agree to the use of cookies."
+#~ msgstr ""
+#~ "Diese Website verwendet Cookies. Wenn Sie sich weiterhin auf dieser Seite "
+#~ "aufhalten, gehen wir von Ihrem Einverständnis aus."
+
+#~ msgid "Learn more"
+#~ msgstr "Mehr Infos"
```
Comment (sun) [inline languages/core-standards-de_DE.po]: These should be removed.

File: templates/cookie-notice.php [hunk 2]
```diff
@@ -5,8 +5,7 @@
 
 <div class="cookie-notice" id="cookie-notice" role="banner">
   <div class="cookie-notice__body">
-    <?= __('This website uses cookies to give you the best possible service. By continuing to use the website, you agree to the use of cookies.', Plugin::L10N) ?>
-    <a class="cookie-notice__link" href="<?= apply_filters('core_standards/cookie_notice/policy_link', '/privacy') ?>" target="_self"><?= __('Learn more', Plugin::L10N) ?></a>
+    <?= sprintf(__('To give you the best possible service, this website uses cookies. By continuing to use it, you agree to our <a href="%s" target="_self">privacy policy</a>.', Plugin::L10N), apply_filters('core_standards/cookie_notice/policy_link', '/privacy')); ?>
```
Comment (sun) [inline templates/cookie-notice.php]: The target is superfluous (_self is the default).

```suggestion
    <?= sprintf(__('To give you the best possible service, this website uses cookies. By continuing to use it, you agree to our <a href="%s">privacy policy</a>.', Plugin::L10N), apply_filters('core_standards/cookie_notice/policy_link', '/privacy')); ?>
```

The class has been removed already, but the string translations were not updated yet.

**Review by sun — CHANGES_REQUESTED** [review]

File: languages/core-standards-de_DE.po (hunk 1 above)
Comment (juanlopez4691) [reply languages/core-standards-de_DE.po]: Fixed.

File: templates/cookie-notice.php (hunk 2 above)
Comment (juanlopez4691) [reply templates/cookie-notice.php]: Ack! Sorry. Will do.

Comment (colourgarden) [reply templates/cookie-notice.php]: We can remove the class but we still need the styling applied. If we're going this route then we also need to update the CSS. [This block](https://github.com/netzstrategen/wordpress-core-standards/blob/master/assets/styles/cookie-notice.scss#L21-L33) will need to be nested inside the block above it using an `a` element selector.

**Comment by juanlopez4691** [comment]
@sun Removing it makes the site styling for links to be applied and in WOHN this doesn't look good: dark red on dark background. Also, rollover colour for both BEGA/WOHN is dark grey. I restored the link class `cookie-notice__link` so that link is always legible.

**Comment by sun** [comment]
> Removing it makes the site styling for links to be applied and in WOHN this doesn't look good: dark red on dark background. Also, rollover colour for both BEGA/WOHN is dark grey. I restored the link class `cookie-notice__link` so that link is always legible.

@juanlopez4691 Did you adjust the CSS as suggested in https://github.com/netzstrategen/wordpress-core-standards/pull/32#discussion_r287998666?

**Comment by juanlopez4691** [comment]
> @juanlopez4691 Did you adjust the CSS as suggested in #32 (comment)?

@sun missed that comment, sorry. Fixed.

File: assets/styles/cookie-notice.scss [hunk 3]
```diff
@@ -16,27 +16,22 @@
...
+      color: currentColor;
+      text-decoration: underline;
+    }
 
-  &:hover,
-  &:focus {
-    opacity: 0.8;
+    &:hover,
+    &:focus {
+      opacity: 0.8;
```
Comment (sun) [inline assets/styles/cookie-notice.scss]: Not sure why we are fiddling with the opacity here. Might also be a remnant of the former button styling that is unnecessary for a text link now.

File: languages/core-standards-de_DE.po (hunk 1 above)
Comment (sun) [reply languages/core-standards-de_DE.po]: Still visible here? Did you commit the change?

**Review by sun — APPROVED** [review]

**Comment by juanlopez4691** [comment]
@sun fixed.

File: assets/styles/cookie-notice.scss (hunk 3 above)
Comment (colourgarden) [reply assets/styles/cookie-notice.scss]: This styling is required and wasn't a leftover from the previous cookie notice.

It's to provide a hover state without relying on a particular text color. For example, if the color of the notice was changed to `#000` for a particular client site, there would still be a suitable hover state without having to specify another color.

**Pull request merged**